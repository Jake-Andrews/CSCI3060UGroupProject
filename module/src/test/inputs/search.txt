Whitby
300.0
2