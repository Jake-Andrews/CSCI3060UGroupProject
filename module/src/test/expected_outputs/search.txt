Welcome, please login to use the program. 
To exit type exit, To start type login: 
Please enter a valid username: 
You have been logged in!

Please enter a command!
For a list of commands type 'help': Please enter the city where the unit you are renting is: 
Please enter a valid price to rent the unit at (max is 999.99): 
Please enter the number of bedrooms (max is 9): 
rentID: 113142f7
Username: USER
City: Whitby
numberOfBedrooms: 2
rentalPricePerNight: 300.00
rentalFlag: false
numberOfNightsRemaining: 0

Daily transaction file string: 04_USER___________AA_113142f7_Whitby____________________2_300.00_00

Please enter a command!
For a list of commands type 'help': Daily transaction file string: 00_USER___________AA____________________________________0_0.00___00

Printing Daily Transactions:04_USER___________AA_113142f7_Whitby____________________2_300.00_00
00_USER___________AA____________________________________0_0.00___00
You have been successfully logged out!
Welcome, please login to use the program. 
To exit type exit, To start type login: Thank you for using our program!